pandas-stubs/core/frame.pyi

```diff
@@ -1096,7 +1096,7 @@ class DataFrame(NDFrame, OpsMixin):
         f: Callable[..., ListLikeExceptSeriesAndStr | Series],
         axis: AxisTypeIndex = ...,
         raw: _bool = ...,
-        result_type: Literal[None] = ...,
+        result_type: None = ...,
```

Comment:
This does not necessarily have to be `Literal[None]`, just `None` is OK, and so are others.